### ISSUE TITLE
Persistent form

### ISSUE THREAD

**Issue by HUGE-D1CK-10-INCHES** [protonemedia/laravel-splade#186]
Is it possible to have a persistent form or do I need to wrap everything into layout?

When uploading large files, I would love to let users run around and do other stuff while files are being uploaded in the background.

Thanks buddy

**Comment by prdpspkt** [comment]
Try Teleportation

**Comment by pascalbaljet** [comment]
@HUGE-D1CK-10-INCHES Do you mean that the upload keeps going while users navigate to other pages?

**Comment by HUGE-D1CK-10-INCHES** [comment]
@pascalbaljet yep, since uploading may take up to hour or two, would like to keep it in the background. I mean someone can open a new tab but would be nicer if it could stay in the background since I will put website into webview for phones, where new tab would not work.. Haven't tried teleportation yet

**Comment by pascalbaljet** [comment]
Got it. Teleportation won’t work. I think you need the Persistent Layout feature but I’ll look into it.

**Comment by HUGE-D1CK-10-INCHES** [comment]
Thanks budd, I thought about teleporting the form into the persistent layout part before submitting the form so it stays there, however if we could make elements persistant directly, would be nicer
